airflow/dags/payments_views/sbmtd_transactions.sql

```diff
@@ -50,17 +50,48 @@ micropayment_device_transactions_dedupe AS (
 /*
   All the unique routes and stops across SBMTD
 */
-route_stops as (
+dated_route_stops as (
     select distinct
         route_id,
-        stop_id
-    from gtfs_schedule.stop_times as st
-    join gtfs_schedule.trips as t using (calitp_itp_id, calitp_url_number, trip_id)
-    join gtfs_schedule.routes as r using (calitp_itp_id, calitp_url_number, route_id)
+        stop_id,
+        case
+          when ((st.calitp_extracted_at >= t.calitp_extracted_at) and
+            (st.calitp_extracted_at >= r.calitp_extracted_at))
+            then st.calitp_extracted_at
+          when ((t.calitp_extracted_at >= st.calitp_extracted_at) and
+            (t.calitp_extracted_at >= r.calitp_extracted_at))
+            then t.calitp_extracted_at
+          when ((r.calitp_extracted_at >= st.calitp_extracted_at) and
+            (r.calitp_extracted_at >= t.calitp_extracted_at))
+            then r.calitp_extracted_at
+        end
+        as calitp_extracted_at,
```

Comment:
[`greatest`](https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#greatest) should work here, right?